openshift-kube-apiserver/admission/autoscaling/managementcpusoverride/admission.go

```diff
@@ -0,0 +1,236 @@
+package managementcpusoverride
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/apiserver/pkg/admission/initializer"
+	"k8s.io/client-go/informers"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/klog/v2"
+	coreapi "k8s.io/kubernetes/pkg/apis/core"
+	"k8s.io/kubernetes/pkg/kubelet/cm"
+)
+
+const (
+	PluginName                 = "autoscaling.openshift.io/ManagementCPUsOverride"
+	podWorkloadLabel           = "workload.openshift.io"
+	managedCapacityLabel       = "workload.openshift.io/cores"
+	containerCPUsAnnotation    = "io.openshift.workload.management.cpu"
+	containerCPUSetAnnotation  = "io.openshift.workload.management.cpuset"
+	namespaceAllowedAnnotation = "workload.openshift.io/allowed"
```

Comment:
I commented in the enhancement already. I don't like to see these annotations and labels without some "internal" infix. Everything you define above becomes an API that we never get rid of again. Let's not do this.

Reply:
Clarification from @smarterclayton that these are meant to be public API, in contrast to the early discussions that we will document that they are not going to be public for 3rd-party use.